doc/howto/import_machines_to_instances.md

```diff
@@ -32,8 +32,22 @@ The tool can create both containers and virtual machines:
   It is also not possible to create a virtual machine from the physical machine that you are using to do the migration, because the migration tool would be using the disk that it is copying.
   Instead, you could provide a bootable image, or a bootable partition or disk that is currently not in use.
 
+The tool can also inject the required drivers into the image:
+
+* To convert the image into raw format and inject the VIRTIO drivers during the conversion, use the following command:
+
+      lxd-migrate --conversion=format,virtio
+
+  ```{note}
+  The conversion option `virtio` requires `virt-v2v-in-place` to be installed on the host where the LXD server runs.
+  ```
+
+* For converting Windows images, you must install additional drivers on the host:
+   * Install the `virtio-win` package or download the [`virtio-win.iso`](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.iso) file and place it in the `/usr/share/virtio-win` directory.
+   * Download `pnp_wait.exe` and `rhsrvany.exe` and place them in the `/usr/share/virt-tools/` directory.
+
    ````{tip}
-   If you want to convert a Windows VM from a foreign hypervisor (not from QEMU/KVM with Q35/`virtio-scsi`),
+   If you want to convert a Windows VM from a foreign hypervisor manually (not from QEMU/KVM with Q35/`virtio-scsi`),
```

Comment:
@ru-fu 

The driver installation section (`Install the virtio-win package or download...`) repeats a few lines lower in the tip (`If you want to convert a Windows VM from a foreign hypervisor maunaly...`).

Could I just say something like this in the tip, to reduce the unnecessary repetition?

Tip title: `Expand to see how to integrate the required drivers to your Windows VM manually`
Tip content:
```
If you want to convert a Windows VM from a foreign hypervisor manually (not from 
QEMU/KVM with Q35/`virtio-scsi`), in addition to the required Windows drivers, you
must also install `virt-v2v`.

Using `virt-v2v`, you can convert images from a foreign hypervisor to `raw` images for
LXD and include the required drivers:

Example 1: ... "Rest of the content remains the same"
```

Reply:
Good catch!

Yes, it makes sense to remove that part from the tip. Just make it very clear that you still need to follow the instructions above - something like ``If you want to convert a Windows VM from a foreign hypervisor manually (not from QEMU/KVM with Q35/`virtio-scsi`), you must install both the required Windows drivers (as described above) and `virt-v2v`.``